operations/app/src/modules/common/private_endpoint/main.tf

```diff
@@ -40,7 +40,7 @@ locals {
   }
 
   option        = local.options[var.type] # Make options a little easier to reference
-  endpoint_name = "${var.name}-${var.type}"
+  endpoint_name = "${var.name}-${var.type}-${substr(sha1(var.endpoint_subnet_id), 0, 8)}"
```

Comment:
The subnet id was added to the endpoint name to avoid naming collisions with setting up private endpoints in both the existing VNET and the Palo Alto VNET.